src/pycram/ontology/ontology.py

```diff
@@ -30,53 +28,58 @@ class OntologyManager(object, metaclass=Singleton):
     Singleton class as the adapter accessing data of an OWL ontology, largely based on owlready2.
     """
 
-    def __init__(self, main_ontology_iri: str = "", ontology_search_path: str = ""):
+    def __init__(self, main_ontology_iri: str = None, ontology_search_path: str = ""):
```

Comment:
also set ontology_search_path to None by default and mark the types as Optional[str]